client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -384,6 +395,21 @@ export const SeoForm = React.createClass( {
 		this.setState( { showPreview: false } );
 	},
 
+	getConflictingSeoPlugins( activePlugins ) {
+		const conflictingSeoPlugins = [
+			'Yoast SEO',
+			'Yoast SEO Premium',
+			'All In One SEO Pack',
+			'All in One SEO Pack Pro',
+		];
+
+		const plugins = flatMap( activePlugins, plugin => (
+			{ name: get( plugin, 'name', '' ), slug: get( plugin, 'slug', '' ) }
+		) );
+
+		return plugins.filter( plugin => includes( conflictingSeoPlugins, plugin.name ) );
+	},
```

Comment:
That's neat, refactored as suggested in ce375ceb671cf73efb60083ee57c2e38e4405ff1. Thanks!